deploy/deploy-jellyfish.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+ARGV_DIRECTORY="$1"
+set -u
+
+[[ "${DEBUG}" == "false" ]] || set -x
+
+pushd $ARGV_DIRECTORY
+
+echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc
```

Comment:
It surprises me that the variable expansion is happening without the double quotes... However, I would still surround the string with double quotes to keep consistency and to enforce good practices :blush: